axon-framework/modules.md

```diff
@@ -102,6 +102,8 @@ Besides main modules, there are several extension modules which complement Axon
 | [Axon CDI](modules.md#axon-cdi)                                                           | axon-cdi                             | org.axonframework.extensions.cdi         | [available](https://search.maven.org/search?q=a:axon-cdi)                             |     [available](https://github.com/AxonFramework/extension-cdi)     |
 | [Axon JGroups](modules.md#axon-jgroups)                                                   | axon-jgroups                         | org.axonframework.extensions.jgroups     | [available](https://search.maven.org/search?q=a:axon-jgroups)                         |   [available](https://github.com/AxonFramework/extension-jgroups)   |
 | [Axon JGroups Spring Boot Starter](modules.md#axon-jgroups-spring-boot-starter)           | axon-jgroups-spring-boot-starter     | org.axonframework.extensions.jgroups     | [available](https://search.maven.org/search?q=a:axon-jgroups-spring-boot-starter)     |   [available](https://github.com/AxonFramework/extension-jgroups)   |
+| [Axon JobRunr Pro](modules.md#axon-jobrunrpro)                                            | axon-jobrunrpro                      | org.axonframework.extensions.jobrunrpro  | [available](https://search.maven.org/search?q=a:axon-jobrunrpro)                      | [available](https://github.com/AxonFramework/extension-jobrunrpro)  |
```

Comment:
Curious: What's the trailing `[available]` for?

Reply:
It's just to have a short link IMHO, not sure if there is something better. It does show the available artifact.

Reply:
Aah, the indentation was off on my screen, so I literally just saw `[available]` :face_exhaling: 
So, nevermind my comment and thanks for clarifying!